src/Haskell/Docs/Cabal.hs

```diff
@@ -29,22 +33,16 @@ getGhcOpsPackageDB gs = map (SpecificPackageDB . trim) pkgDBOps
 
 -- | Get all installed packages, filtering out the given package.
 getAllPackages :: [String] -> Ghc [PackageConfig.PackageConfig]
-getAllPackages gs =
+getAllPackages _gs =
   do flags <- getSessionDynFlags
      return (fromMaybe [] (pkgDatabase flags))
-  -- do config <- configureAllKnownPrograms
```

Comment:
It was already commented out, so I just removed the useless comment.

I can put that comment back in if you'd prefer.